src/main/scala/scalafix/sbt/ScalafixPlugin.scala

```diff
@@ -395,13 +436,15 @@ object ScalafixPlugin extends AutoPlugin {
         scalafixHelp
       } else {
         val scalafixConf = (config / scalafixConfig).value.map(_.toPath)
+        val resolvers =
+          ((ThisBuild / scalafixAdaptSbtResolvers).value ++ (ThisBuild / scalafixResolvers).value).distinct
```

Comment:
Since ordering matters for resolution (and `.distinct` preserves it I believe), favor the key meant to be set/overriden by the user, for more control.
```suggestion
          ((ThisBuild / scalafixResolvers).value ++ (ThisBuild / scalafixAdaptSbtResolvers).value).distinct
```

Reply:
agree